### PR TITLE
Sync coin assets & bump release version

### PR DESCRIPTION
**Pull request by takenagain**
- Sync config files 
- Sync icons
- Update bundled commit hash
- Update wallet only coins list
- Bump the version number to 0.9.1

### CODE REVIEW

File: lib/app_config/app_config.dart
Comment (CharlVS) [inline lib/app_config/app_config.dart:1]: Excellent attention to detail for reconciling wallet-only coins with the current status in the coins config and removing those that no longer exist. There may be a reason we're not aware of why they were added as wallet-only. Please test one of the updated coins with a test swap and/or go back in the git history of the coins config file and check if there is a log of `"wallet_only": true` being changed to `wallet_only": false`.

Comment (CharlVS) [reply lib/app_config/app_config.dart:1]: E.g. `ETH-ARB20`

**Review by smk762 — APPROVED** [review]
Working as expected, thanks. I was able to cast my vote via qr code

File: lib/app_config/app_config.dart
Comment (takenagain) [reply lib/app_config/app_config.dart:1]: I've re-ordered the list alphabetically, which has made the diff a bit unclear. So, I've summarized the changes below for clarity.

### Unknown: `wallet_only` changed to false or removed

#### ETH-ARB20
@smk762, should `ETH-ARB20` be added back to the wallet-only-coins list? I noticed that the `ETH-ARB20` protocol coins (`CELR-ARB20`, and `USDT-ARB20`) are wallet only, but not `ETH-ARB20`. I could also not find `ETH-ARB20` or related coins in the coins list for activation.

#### SUPERNET
The coin has been activated and swapped successfully (KMD -> SUPERNET).

### Removed from `coins.json`

- BET
- BOTS
- CRYPTO
- HODL
- JUMBLR
- LUNA-ERC20
- LUNA-BEP20
- LUNA-HRC20
- LUNA-PLG20
- MSHARK
- ETHK-OPT20
- ETHR-ARB20
- MGW
- DEX
- PANGEA
- REVS
- UST-ERC20
- UST-BEP20
- UST-HRC20
- UST-PLG20
- XVC-OLD

### Unchanged (`"wallet_only": true`)

- RBTC
- OSMO
- NVC
- XPM
- SXP-BEP20
- SXP-ERC20
- VOTE2023
- GALA-BEP20
- BBK
- RDD
- PINK
- POT

Comment (cipig) [reply lib/app_config/app_config.dart:1]: > should `ETH-ARB20` be added back to the wallet-only-coins list? I noticed that the `ETH-ARB20` protocol coins (`CELR-ARB20`, and `USDT-ARB20`) are wallet only, but not `ETH-ARB20`. I could also not find `ETH-ARB20` or related coins in the coins list for activation.

swaps with ARB20 don't work: https://github.com/KomodoPlatform/komodo-defi-framework/issues/1079#issuecomment-1773766932 ... i guess that ETH-ARB20 is not wallet-only in order to be able to test future changes that eventually lead to a working swap

**Review by CharlVS — APPROVED** [review]

**Pull request merged**